src/controlCenter/widgets/menubar/menubar.vala

```diff
@@ -92,14 +94,19 @@ namespace SwayNotificationCenter.Widgets {
 
                     Gtk.Box menu = new Gtk.Box (Gtk.Orientation.VERTICAL, 0);
                     menu.get_style_context ().add_class (obj.name);
```

Comment:
Forgot to mention this before, we should avoid setting the class of the menu and the buttons if the `obj.name` is null. My bad 😅